tests/TestLeibnitz.elm

```diff
@@ -0,0 +1,105 @@
+module TestLeibnitz exposing (..)
+
+import Expect exposing (Expectation)
+import Formula exposing (Formula(..))
+import Formula.Parser
+import Formula.Signed exposing (Signed(..))
+import Term exposing (Term(..))
+import Tableau exposing (..)
+import Test exposing (..)
+import Validate
+import Validation.Common
+import Validation.Rules.Leibnitz
```

Comment:
`Validate` and `Validation.Common` are not used. You could expose `replaceTermWithVar` the content of `Validation.Rules.Leibnitz` to make the code below less verbose.